### DISCUSSION TITLE
Is there a way to cleanup the deleted notes?

### DISCUSSION THREAD

**Discussion by fooyoo** [TriliumNext/Trilium#3258] [General]
I'd like to share an empty DB to others, just including scripts/theme/css/templates and some note trees/frame. I deleted all my private notes and found them can be seen in "recent note" history. how to clean  them fully?

**Comment by XXXJetfireXXX** [comment]
There's no need to manually delete your personal data in order to do that. If you want to share just the scripts and structure of your DB, use anonymization instead:
![image](https://user-images.githubusercontent.com/43646178/198830254-351a8f9a-d90c-4c17-a6ab-a6e8e14eab24.png)

In other cases, if you want to erase all deleted notes, you can do it here:
![image](https://user-images.githubusercontent.com/43646178/198830487-9a1d8522-d0da-4418-ad32-7cb4a9708ba8.png)
Be careful though, as you won't be able to recover the data that was completely removed. (You'll have to use backups)

**Comment by fooyoo** [comment]
Thank you very much for so the detailed guidance, I am too careless, didn't find they are there:).
